langwatch/next.config.mjs

```diff
@@ -125,6 +125,13 @@ const config = {
     // eslint-disable-next-line @typescript-eslint/dot-notation
     config.resolve.alias["zod"] = `${__dirname}/node_modules/zod`;
 
+    // Add fallback for pino logger requirements
+    config.resolve.fallback = {
+      ...config.resolve.fallback,
+      worker_threads: false,
+      fs: false,
+    };
```

Comment:
I'm concerned if this could end up inadvertely affecting other things in the future, for example maybe we import a nodejs module we shouldn't on the frontend, but it doesn't fail as it should because those fallbacks are in place